mne/io/brainvision/brainvision.py

```diff
@@ -55,7 +55,7 @@ class RawBrainVision(_BaseRaw):
         triggers will be ignored. Default is 0 for backwards compatibility, but
         typically another value or None will be necessary.
     event_id : dict | None
-        The id of the event to consider. If None (default),
+        The id of special events to consider. If None (default),
```

Comment:
I mean "special" in the sense of "Not 'S100' or 'R100'". Hm. I guess I'll have to write a few more lines. It should be clear that it's for non-standard (trigger channel) events.